src/Connection.php

```diff
@@ -114,13 +114,31 @@ public function getSchemaBuilder()
     /**
      * Get the MongoDB database object.
      *
+     * @deprecated Use getDatabase() instead
+     *
      * @return Database
      */
     public function getMongoDB()
     {
         return $this->db;
     }
 
+    /**
+     * Get the MongoDB database object
+     *
+     * @param string|null $name Name of the database, or null to get the default one
+     *
+     * @return Database
+     */
+    public function getDatabase(?string $name = null): Database
+    {
+        if ($name === null) {
+            return $this->db;
+        }
+
+        return $this->connection->{$name};
```

Comment:
Would it be preferable to explicitly invoke `Client::getDatabase()` here instead of going through `__get()`, which will do so on its own?

Reply:
This was to ensure compatibility before the upgrade to mongodb/mongodb v1.21